lib/auth.js

```diff
@@ -254,7 +254,7 @@ const authClient = new Model({
             },
           };
 
-          return apiClient.put('/../users', data, config.jsonHeaders)
+          return makeCredentialHTTPRequest('PUT', '/../users', data, config.jsonHeaders)
```

Comment:
We need to supply the API domain + path URL here as this defaults to the window origin which isn't the API host. 
```suggestion
          const url = config.host + '/users';
          return makeCredentialHTTPRequest('PUT', 'url', data, config.jsonHeaders)
```

Reply:
Good catch!